internal/handlers/pg/msg_update.go

```diff
@@ -160,15 +160,20 @@ func (h *Handler) MsgUpdate(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		matched += int32(len(resDocs))
 
 		for _, doc := range resDocs {
-			if err = common.UpdateDocument(doc, u); err != nil {
-				return nil, lazyerrors.Error(err)
+			var changed bool
```

Comment:
Let’s change this declaration to `:=` to make all declarations the same inside `for` block.

Reply:
done.

Reply:
But it looks like nothing changed. 😃

Reply:
sorry, updated.